content/techniques/validation.md

```diff
@@ -252,6 +252,112 @@ findOne(
 
 > info **Hint** The `ParseIntPipe` and `ParseBoolPipe` are exported from the `@nestjs/common` package.
 
+#### Mapped types
+
+As you build out features like **CRUD** (Create/Read/Update/Delete) it's often useful to construct variants on a base entity type. Nest provides several utility functions that perform type transformations to make this task more convenient.
+
+> **Warning** If you are using the `@nestjs/swagger` package see [here](/openapi/mapped-types), while if you use the `@nestjs/grapqhl` package see [here](/graphql/mapped-types)
+
+##### Partial
+
```

Comment:
```suggestion
> **Warning** If your application uses the `@nestjs/swagger` package, see [this chapter](/openapi/mapped-types) for more information about Mapped Types. Likewise, if you use the `@nestjs/grapqhl` package see [this chapter](/graphql/mapped-types). Both packages heavily rely on types and so they require a different import to be used. Therefore, if you used `@nestjs/mapped-types` (instead of an appropriate one, either `@nestjs/swagger` or `@nestjs/graphql` depending on the type of you app), you may face various, undocumented side-effects.

##### Partial

```